src/adapter/src/coord.rs

```diff
@@ -3243,6 +3247,7 @@ pub fn serve(
         controller_envd_epoch,
         controller_txn_wal_tables,
         storage,
+        persist_client,
```

Comment:
We do already have a `PersistClientCache` and a `PersistLocation`, so we can open a new client instead of plumbing an existing one. I just updated the PR to do that.